Lombiq.Hosting.Azure.ApplicationInsights/Extensions/ApplicationInsightsInitializerExtensions.cs

```diff
@@ -38,7 +38,15 @@ public static class ApplicationInsightsInitializerExtensions
 
         services.Configure<TelemetryConfiguration>(config =>
         {
-            if (!applicationInsightsOptions.EnableLocalDevelopment)
+            if (applicationInsightsOptions.EnableSecureLocalDevelopment)
```

Comment:
We need to support the scenario of Live Metrics receiving data without a secure control channel (since you only need that for control, not for ingestion). Just configuring the connection string should still work.

Then, if you want to secure the control channel, you can optionally do that with a) API key (deprecated), b) secrets with a Service Principal or c) with a managed identity when deployed to Azure. Note that while we need b) for local _development_ too, it's actually not just for that, since you can use Service Principals in Azure too (you may actually need that for services not supporting Managed Identity).

Both b) and c) satisfy Local Authentication being disabled. Right? Or is b) still Local Authentication?

Do I understand correctly that under https://github.com/Lombiq/Orchard-Azure-Application-Insights/pull/85#discussion_r1627908324 you mention that with Local Authentication disabled you need to use a Service Principal or Managed Identity even if you don't want to secure the control channel?